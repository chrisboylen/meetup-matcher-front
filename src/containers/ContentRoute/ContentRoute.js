import React from 'react';
import { Route, withRouter, Switch } from 'react-router-dom';
import { Home } from '../../components/Home/Home';
import Login from '../Login/Login';
import Signup from '../Signup/Signup';
import User from '../User/User';

export const ContentRoute = () => {
  return (
    <Switch>
      <Route exact path="/" component={Home} />
      <Route exact path="/login" component={Login} />
      <Route exact path="/signup" component={Signup} />
      <Route exact path="/user" component={User} />
    </Switch>
  );
};

export default withRouter(ContentRoute);