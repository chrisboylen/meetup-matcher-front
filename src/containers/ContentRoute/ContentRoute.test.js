import React from 'react';
import { MemoryRouter } from 'react-router';
import { shallow, render} from 'enzyme';
import { ContentRoute, mapStateToProps } from './ContentRoute';

describe('ContentRoute', () => {
  it('should match snapshot with default path', () => {
    const wrapper = shallow(<ContentRoute />);
    
    expect(wrapper).toMatchSnapshot();
  });

 
});


