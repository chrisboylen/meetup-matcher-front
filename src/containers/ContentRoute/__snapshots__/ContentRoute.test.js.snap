// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentRoute should match snapshot with default path 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/signup"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/user"
  />
</Switch>
`;
